modules/mtbseq/mtbseq_cohort.nf

```diff
@@ -0,0 +1,44 @@
+// NOTE: To properly setup the gatk inside the docker image
+// - Download the gatk-3.8.0 tar file from here https://console.cloud.google.com/storage/browser/gatk-software/package-archive/gatk;tab=objects?prefix=&forceOnObjectsSortingFiltering=false
+// - tar -xvf GATK_TAR_FILE
+// - gatk-register gatk_folder/gatk_jar
+
+nextflow.enable.dsl = 2
+
+params.results_dir = "${params.outdir}/mtbseq/cohort"
+params.save_mode = 'copy'
+params.should_publish = true
+
+process MTBSEQ_COHORT {
+    publishDir params.results_dir, mode: params.save_mode, enabled: params.should_publish
+    // TODO port to errorStrategy and maxRetries
+    // validExitStatus 0, 1, 2
```

Comment:
@Mxrcon ,  this would be a good one if you try to tackle.

Reply:
ok! I'll try to give it a try, :fearful: worried about my approaches to it.